llvm/include/llvm/ExecutionEngine/JITLink/aarch64.h

```diff
@@ -603,6 +767,26 @@ inline Error applyFixup(LinkGraph &G, Block &B, const Edge &E) {
     *(ulittle32_t *)FixupPtr = FixedInstr;
     break;
   }
+  case GotPageOffset15: {
+    Section *GOTSection =
+        G.findSectionByName(aarch64::GOTTableManager::getSectionName());
```

Comment:
In other backends we've introduced a `GOTSymbol` parameter to `applyFixup` that allows linkers to cache this symbol (see e.g. x86_64.h) . I think that it would be reasonable to do that here too. This should also improve performance, since you can cache the section start symbol (which you should get from `SectionRange`).